janusgraph-solr/src/test/java/org/janusgraph/diskstorage/solr/SolrIndexTest.java

```diff
@@ -158,15 +156,18 @@ public void testSupport() {
      * Dropping collection is not implemented with Solr Cloud to accommodate use case where collection is created
      * outside of JanusGraph and associated with a config set with a different name.
      */
-    @Override @Test @Ignore
+    @Override @Test @Disabled
     public void clearStorageTest() throws Exception {
         super.clearStorageTest();
     }
 
-    @Test(expected = IllegalArgumentException.class)
+    @Test
     public void testMapKey2Field_IllegalCharacter() {
-        KeyInformation keyInfo = new StandardKeyInformation(Boolean.class, Cardinality.SINGLE);
-        index.mapKey2Field("here is an illegal character: •", keyInfo);
+        assertThrows(IllegalArgumentException.class, () -> {
+
+            KeyInformation keyInfo = new StandardKeyInformation(Boolean.class, Cardinality.SINGLE);
+            index.mapKey2Field("here is an illegal character: •", keyInfo);
+        });
```

Comment:
After reading @porunov 's comments and rereading the docs and [user guide](https://junit.org/junit5/docs/current/user-guide/#writing-tests-assertions) I think this should be rewritten like this:
```
Throwable exception = assertThrows(IllegalArgumentException.class, () -> {
    KeyInformation keyInfo = new StandardKeyInformation(Boolean.class, Cardinality.SINGLE);
        index.mapKey2Field("here is an illegal character: •", keyInfo);
    });
    assertEquals("here is an illegal character: •", exception.getMessage());
```

Reply:
I can add a assert to check the exception.This check didn't exists before. I want to change the flow  test as less as possible.

Reply:
@chupman , I agree here with @farodin91 . I think we shouldn't change the flow of tests in this PR.